tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "rootDir": "./src",
     "lib": ["esnext", "dom"],
-    "allowJs": true,
+    "allowJs": false,
```

Comment:
Would be cool to do it, but we are in the middle of TS migration, so we have plenty of JS files :(
Let's revert for now

Reply:
I thought Babel should be capable of running the JS and TS, the tsconfig throws this overwriting error if true. If this issue will resolve itself then I can leave this alone.
 `Cannot write file 'c:/Code/OpenSource/vibe-storybook-components/src/components/component-rule/component-rule-constants.js' because it would overwrite input file.`